code/datums/wires/wires.dm

```diff
@@ -117,7 +117,7 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 			var/obj/item/I = L.get_active_hand()
 			holder.add_hiddenprint(L)
 			if(href_list["cut"]) // Toggles the cut/mend status
-				if(I.is_wirecutter())
+				if(I?.is_wirecutter())
```

Comment:
Wait that's valid syntax?

Reply:
The byond docs say `This is used to access the procs and vars of a prototyped object. It is just like the . operator, but if the object is null, the access does not happen and there will be no runtime error.` and `When reading A?.B, it's equivalent to A && A.B except that A is only evalulated once, even if it's a complex expression like a proc call.`

Reply:
Null-safe member access operator. I was impressed that BYOND added it already, every language ought to add it.  In a situation like this,  doing  `I && I.is_wirecutter()` would have been just fine too.    But it really is cool in cases when getting an actual value rather than just being a boolean predicate, or when the left hand side is costly.

Reply:
_cackling over the potential loss of characters typed and most likely usually exceedingly minor performance improvements_